ext/standard/random.c

```diff
@@ -205,7 +205,7 @@ PHP_FUNCTION(random_bytes)
 	ZEND_PARSE_PARAMETERS_END();
 
 	if (size < 1) {
-		zend_throw_exception(zend_ce_error, "Length must be greater than 0", 0);
+		zend_throw_exception_ex(zend_ce_error, 0, "Argument 1 must be greater than 0, %lld given", size);
```

Comment:
OK, this is not a type error, but these kind of messages could also be migrated.